scripts/checkpoint_converters/convert_gpt_nemo_to_mcore.py

```diff
@@ -40,22 +40,22 @@
 
 Then, run this conversion script:
 python convert_gpt_nemo_to_mcore.py \
- --input_name_or_path <path to extracted, TP1 PP1 legacy checkpoint folder> \
- --output_path <path to output nemo file>
+ --input-name-or-path <path to extracted, TP1 PP1 legacy checkpoint folder> \
+ --output-path <path to output nemo file>
 """
 
 
 def get_args():
     parser = ArgumentParser()
     parser.add_argument(
-        "--input_name_or_path",
+        "--input-name-or-path",
         type=str,
         default=None,
         required=True,
         help="Path to extracted, TP1 PP1 NeMo GPT checkpoint.",
     )
     parser.add_argument(
-        "--output_path",
+        "--output-path",
```

Comment:
can you don't change this to dash, keep it as underscore

Reply:
See my response in https://github.com/NVIDIA/NeMo/pull/9628#discussion_r1691806797